language/sample-apps/text-predict-cloudfunction/requirements.txt

```diff
@@ -0,0 +1,3 @@
+functions-framework==3.*
+google-cloud-aiplatform==1.27.1
```

Comment:
Could you also try to update the version here to at least 1.31.0?

Reply:
Done